### PR TITLE
Add InfoPanel component

### PR DESCRIPTION
**Pull request by TejasQ**
### Summary
This PR adds a component that relays relevant information to a user inside of a card, and is essential in order to implement the following part of the design.

image

### To be tested

Tester 1

- [ ] No error/warning in the console
- [ ] The netlify build is working
  

Tester 2

- [ ] No error/warning in the console
- [ ] The netlify build is working

### CODE REVIEW

File: src/CardInfo/CardInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import * as React from "react"
+
+import Spinner from "../Spinner/Spinner"
+import styled from "../utils/styled"
+
+export interface CardInfoProps {
```
Comment (peterszerzo) [inline src/CardInfo/CardInfo.tsx]: Add `extends DefaultProps`

**Comment by peterszerzo** [comment]
@TejasQ a few things per image below:
* `running -> loading` to make it sound a bit more generic.
* spinner should center in this simplest use-case (nudging the `top` value is fine).
* it doesn't seem like including `Card` in the name adds much content (everything is in a card, this is also in a card..). Would simply `Info` or `InfoPanel` make sense?

screen shot 2018-08-01 at 4 08 23 pm

**Comment by peterszerzo** [comment]
Super cool - one last thing, can we have a bit more space between the spinner and the edge of the text here:

screen shot 2018-08-01 at 4 35 34 pm

(just a few more pixels, no need to match outside padding)

**Comment by TejasQ** [comment]
Please specify a number if you feel strongly about it since this was not in the design and it is currently `theme.space.base`.

**Comment by peterszerzo** [comment]
@TejasQ make it `theme.space.small`

**Comment by TejasQ** [comment]
[Create InfoPanel component for Labs deployment/sync statuses](https://trello.com/c/GY3pN5WR/352-create-infopanel-component-for-labs-deployment-sync-statuses)

**Pull request merged**